kuksa-client/kuksa_client/grpc/__init__.py

```diff
@@ -331,7 +331,7 @@ def from_message(cls, message: types_pb2.Datapoint):
                 timestamp = message.timestamp.ToDatetime(
                             tzinfo=datetime.timezone.utc,
                             )
-            except OverflowError:
+            except ValueError:
```

Comment:
I am by the way looking at how to better integrate logger errors in the CLI - because in this case this error was not visible, right

Reply:
Yes! Nothing was visible from inside the CLI. However, please note that the line you marked is just a mandatory change from the grpc dependency update. The updated version made a test fail (due to a signature change), which was fixed here. 

The real bugfix is this line here:
https://github.com/eclipse-kuksa/kuksa-python-sdk/pull/27/files#diff-2bf17a95c643b27d960c21766e25624dc770b44ee696e3097c5f07d9ecd43525R41

An uncaught exception during json.dumps caused the subscription thread to terminate and no longer processed messages. I don't think that this case is covered by the logger.